### PR TITLE
day alert only

### PR DESCRIPTION
**Pull request by MateoLostanlen**
Our network is not trained to work at night, moreover we have false alerts at daybreak, so I propose to send alerts only between 8am and 9pm

### CODE REVIEW

File: pyroengine/engine/engine.py [hunk 1]
```diff
@@ -90,7 +91,10 @@ def predict(self, frame, pi_zero_id=None):
             if self.use_api and self.event_appening[pi_zero_id]:
                 frame.save(self.stream, format='JPEG')
                 # Send alert to the api
-                self.send_alert(pi_zero_id)
+                now = datetime.now()
+                current_hour = int(now.strftime("%H"))
```
Comment (florianriche) [inline pyroengine/engine/engine.py:95]:
```suggestion
                current_hour = d.hour
```

**Review by florianriche — APPROVED** [review]
I suggested a change but ok otherwise

**Comment by frgfm** [comment]
> Our network is not trained to work at night, moreover we have false alerts at daybreak, so I propose to send alerts only between 8am and 9pm

Thanks for the PR! 
Apart from the code itself:
- we may need a wider time range. Summertime daylight differs a lot from wintertime :sweat_smile: 
- this needs to be set as local time, but the code seems to be comparing UTC timestamps?
- if we don't send alerts during that time, perhaps it might be wise to keep saving pictures but not analyzing them to ease the load of the raspberry PI?

**Comment by Akilditu** [comment]
> > Our network is not trained to work at night, moreover we have false alerts at daybreak, so I propose to send alerts only between 8am and 9pm
> 
> Thanks for the PR!
> Apart from the code itself:
> 
> * we may need a wider time range. Summertime daylight differs a lot from wintertime 😅
> * this needs to be set as local time, but the code seems to be comparing UTC timestamps?
> * if we don't send alerts during that time, perhaps it might be wise to keep saving pictures but not analyzing them to ease the load of the raspberry PI?

Yes agreed on those points as discussed by @pechouc maybe we should consider doing this on the platform side (filtering alerts displayed on an API such as https://sunrise-sunset.org/api) what do you guys think ?

**Comment by frgfm** [comment]
> Yes agreed on those points as discussed by @pechouc maybe we should consider doing this on the platform side (filtering alerts displayed thanks to an API such as https://sunrise-sunset.org/api) what do you guys think ?

Yes I didn't think about it first, but it would be actually much simpler indeed :+1:

**Pull request closed without merging**